docs/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -175,20 +175,16 @@ import * as Sentry from "@sentry/nextjs";
 import NextError from "next/error";
 import { useEffect } from "react";
 
-export default function GlobalError({
-  error,
-}: {
-  error: Error & { digest?: string };
-}) {
+export default function GlobalError({ error }: { error: Error & { digest?: string } }) {
   useEffect(() => {
     Sentry.captureException(error);
   }, [error]);
 
   return (
     <html>
       <body>
-        {/* This is the default Next.js error component but it doesn't allow omitting the statusCode property yet. */}
-        <NextError statusCode={undefined as any} />
+        {/* Passing `0` as `statusCode` as the type has to be a number, but we do not get a status code here. */}
```

Comment:
```suggestion
        {/* `NextError` is the default Next.js error page component. Its type
        definition requires a `statusCode` prop, however, since the App Router
        does not expose status codes for errors, we simply pass 0 to render a
        generic error message.*/}
```

I would put the same text in the wizard PR.